aws/resource_aws_appmesh_mesh_test.go

```diff
@@ -0,0 +1,119 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/appmesh"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsAppmeshMesh_importBasic(t *testing.T) {
+	resourceName := "aws_appmesh_mesh.foo"
+	rName := fmt.Sprintf("tf-test-%d", acctest.RandInt())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAppmeshMeshDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAppmeshMeshConfig(rName),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
```

Comment:
We prefer to put the `ImportState` test step in each acceptance test instead of separate and duplicate testing. 👍